### DISCUSSION TITLE
Why remove the gf.components.bezier?

### DISCUSSION THREAD

**Discussion by AstraArcane** [gdsfactory/gdsfactory#3487] [Q&A] [answered]
As an engineer, I’m pleased to see the active updates of gdsfactory and the enhancements of its functions. However, it seems that the latest version 8.25.0 has removed the attribute `gf.components.bezier`, which has made my old code invalid. And I haven't been able to find a proper alternative.

Bezier curves are common components for my layout, so the update has left me quite confused.

**Comment by joamatab** [comment]
Thank you for your for bringing this to our attention—and apologies for the oversight.

see fix here
https://github.com/gdsfactory/gdsfactory/pull/3488

I can make release a patch for it

**Comment by sebastian-goeldi** [comment]
Just to be clear on @joamatab 's point. It wasn't actually removed. But in order to make gdsfactory work better with IDE integration (in this case type checking and autocompletion), the components folder had to be slightly rearranged. And this was just an oversight when importing them. So, updating after the new release should make it work ahgain.

**Reply by joamatab** [reply, marked as answer]
yes, you can `pip install gdsfactory --upgrade` you should get the 8.25.1 

where bezier is exposed as `gf.components.bezier`
